src/main.ts

```diff
@@ -5,6 +5,8 @@ import {DocumentBuilder, SwaggerModule} from '@nestjs/swagger';
 import {RootModule} from './RootModule';
 
 const port = process.env.PORT || 21080;
+const {name}: {name: string} = require('../package.json');
+const {version}: {version: string} = require('../package.json');
```

Comment:
```suggestion
const {name, version}: {name: string, version: string} = require('../package.json');
```

Reply:
I found the correct syntax to be `const {name, version}: {name: string; version: string} = require('../package.json');` ;)

Reply:
Exactly! :smiley: